src/container/Search/Search.tsx

```diff
@@ -80,18 +78,23 @@ const Search = () => {
   }
 
   const getOptionLabel = (option: any) => {
-    if (isTransaction(option)) {
-      return `${
-        isSmallBreakpoint
-          ? getDisplayShortHash(option.transaction_identifier.hash)
-          : option.transaction_identifier.hash
-      } - Block: ${option.block_identifier?.index}`
+    if (isTransaction(option) && option.blocks) {
+      const mainBlock = option.blocks.find((block) => block.main === true)
+      if (mainBlock) {
+        return `${
+          isSmallBreakpoint
+            ? getDisplayShortHash(option.hash.toUpperCase())
+            : option.hash.toUpperCase()
```

Comment:
Minor but I'd wanna reuse a variable rather than do the operation twice.
```suggestion
        const hash = option.hash.toUpperCase()
        return `${
          isSmallBreakpoint
            ? getDisplayShortHash(hash)
            : hash
```